_pages/model-deployment/xgboost.md

```diff
@@ -164,3 +173,168 @@ If you don't recall from the <a href="{{site.baseurl}}/algorithm-development/alg
+{: .notice-warning}
+
+{% raw %}
+<div ng-controller="GettingStartedControl" ng-init="setCardContent('YOUR_USERNAME')" class="gs-code-container">
+  <div class="code-toolbar ph-16 pv-8">
+    <div class="btn-group dropdown">
+      <button type="button" class="btn btn-default dropdown-toggle gs-dropdown pa-0" data-toggle="dropdown">
+        <div class="lang-logo white-logo mr-4" ng-class="lang"></div>
+        <span ng-bind="languages[lang]" class="mr-4"></span>
+        <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu gs-languages pv-4" role="menu">
+        <li ng-repeat="(language, displayName) in languages" class="mb-0">
+          <a class="caption" ng-click="setLang(language)">
+            <div class="lang-logo color-logo mr-4" ng-class="language"></div>
+            <span ng-bind="displayName"></span>
+          </a>
+        </li>
+      </ul>
+    </div>
+    <button type="button" class="btn btn-flat text-light-primary copy-btn" ng-click="copyCode(lang)">
+      <i class="fa fa-copy"></i>
+    </button>
+  </div>
+
+<!-- PYTHON -->
+  <div class="tab-pane code__pane gs-pane" id="python" ng-show="lang==='python'" ng-cloak><pre class="getting-started-code"><code class="demo-code-sample hljs python">username = <span class="hljs-string">"<span class="hover-info">YOUR_USERNAME<div class="hover-content card pa-16" ng-bind-html="cardContent"></div></span>"</span>
+api_key = <span class="hljs-string">"<span class="hover-info">YOUR_API_KEY<div class="hover-content card pa-16" ng-bind-html="cardContent"></div></span>"</span>
+</code></pre><textarea id="python-copy-text" class="copy-text">
+username = "YOUR_USERNAME"
+api_key = "YOUR_API_KEY"</textarea></div>
+</div>
+{% endraw %}
+
+
+And continue with defining the name of the algorithm you want to create and your local path to clone the repository. An example definition would be:
+
+```python
+algo_name = "xgboost_basic_sentiment_analysis"
+local_dir = "../algorithmia_repo"
+```
+
+Now you can use the utility functions to create the algorithm and clone it on your configured path:
+```python
+algo_utility = algorithmia_utils.AlgorithmiaUtils(api_key, username, algo_name, local_dir)
+algo_utility.create_algorithm()
+algo_utility.clone_algorithm_repo()
+```
+
+### Creating the Algorithm Script and Dependencies
+The folllowing code pieces programmatically creates the algorithm script that handles our requests, and the dependency file that is used when building the container for our algorithm on the Algorithmia environment.
+
+For this we will use the  `%%writefile` macro, but you can always use another editor to edit and save your files.
+
+
+```python
+%%writefile $algo_utility.algo_script_path
+import Algorithmia
+import joblib
+import numpy as np
+import pandas as pd
+import xgboost
+
+model_path = "data://asli/xgboost_demo/musicalreviews_xgb_model.pkl"
+client = Algorithmia.client()
```

Comment:
remind folks to pass in their API key by placing in "YOUR_API_KEY" and replace "data://asli" to "data/YOUR-USERNAME" (we used to pull their username and api key but looks like that doesn't work anymore but it's helpful to have those as placeholders)